scripts/us_census/pep/us_pep_sexrace/postprocess.py

```diff
@@ -0,0 +1,374 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to read all
+the generated csv for
+National, State and County
+and generate final csv, MCF, TMCF file
+"""
+
+import pandas as pd
+import os
+import enum
+
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+
+
+class Outputfiles(enum.Enum):
+    NationalBefore2000 = 1
+    StateCountyBefore2000 = 2
+    StateCountyAfter2000 = 3
+    NationalAfter2000 = 4
+
+
+def write_to_tmcf(filename: str, tmcf: str):
+    with open(_CODEDIR + "/output_files/final/" + filename + ".tmcf",
```

Comment:
I see the directory path is reused in multiple places (e.g., write_to_mcf()), can we abstract it out as a sharable var or function? 

Also, I see other files (preprocess.py) also use the dir path, so maybe put it to a util lib?

Reply:
Abstracted the common paths used and placed them in common file.